allensdk/internal/brain_observatory/roi_filter_utils.py

```diff
@@ -181,9 +182,9 @@ def calculate_max_border(motion_df, max_shift):
         [right_shift, left_shift, down_shift, up_shift]
     '''
     # strip outliers
-    x_no_outliers = motion_df["x"][(motion_df["x"] >= -max_shift) & \
+    x_no_outliers = motion_df["x"][(motion_df["x"] >= -max_shift) &
```

Comment:
Hmm... can you move the operator to the start of the line?

See:
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
done, though this is not part of the standard

Reply:
Weird... maybe our flake8 is not up to date or something?

https://www.flake8rules.com/rules/W504.html

Reply:
My local version was not complaining, and I've made no custom tweaks:
```
$ flake8 --version
3.7.9 (mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 3.6.10 on Linux
```
I think that is Oct 2019